.github/workflows/update_knapsack_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Update Knapsack Report
+
+# This workflow should be scheduled at certain intervals
+on:
+ schedule:
+     - cron: '0 5 31 2 *'    
+# The above cron does not run. Replace with the wanted periodicity.
+# For example, the following configuration would schedule it every 3 months
+#    - cron: '0 0 1 */3 *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:9.4
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      RAILS_ENV: test
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 3.1
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 3.1
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Ruby gems
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+    - name: Setup Database
+      run: bundle exec rails db:create db:migrate
+    - name: Update Knapsack Report
+      run: KNAPSACK=true KNAPSACK_GENERATE_REPORT=true bundle exec rspec
+    - name: Commit files
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add knapsack_rspec_report.json
+        git commit -m "Update knapsack_rspec_report.json"
+    - name: Push changes
+      run: git push origin HEAD:update-knapsack-report
+    - name: Initialize Pull Request
```

Comment:
I'm wondering if would make sense to push directly to master without having to create a PR at all. That's kinda what  the [git-ops](https://www.redhat.com/es/topics/devops/what-is-gitops) is about, so it's probably not so wrong for this use case either

Reply:
We discussed it and we think it better to create a pull request, review it, and then merge. Something similar to what dependabot does.